schema/settings.schema.json

```
@@ -13,6 +13,14 @@
       "type": "object",
       "additionalProperties": true,
       "properties": {
+        "newSearchResultFilters": {
```

May be good to note that the filters themselves haven't changed in terms of what's available, just the way in which we filter. 

```suggestion
        "newSearchResultFiltersPanel": {
          "description": "Enables new experimental search results filters panel"
```